service/src/event/service.rs

```diff
@@ -230,29 +231,75 @@ impl CeramicEventService {
     }
 }
 
-#[derive(Debug, PartialEq, Eq, Default)]
-pub struct InsertResult {
-    pub(crate) store_result: ceramic_store::InsertResult,
-    pub(crate) missing_history: Vec<EventId>,
+#[derive(Debug)]
+pub struct IncomingEvent;
+#[derive(Debug)]
+pub struct InitialIncomingEvent<'a> {
```

Comment:
why have this type at all if it's just a thin wrapper around ReconItem?  Why not just have `IncomingEvent::parsed` take a`ReconItem` as input?

For that matter, why have this `IncomingEvent` type at all?  Why not just a standalone function that takes `ReconItem` and parses it?

Reply:
fair enough. I was trying to quickly add a state machine/builder pattern but it's not worth it yet. just deleted it and put it back to how it was